lbrynet/daemon/Daemon.py

```diff
@@ -1067,12 +1070,18 @@ def jsonrpc_status(self, session_status=False):
         """
 
         # on startup, the wallet or network won't be available but we still need this call to work
-        has_wallet = self.session and self.session.wallet and self.session.wallet.network
-        local_height = self.session.wallet.network.get_local_height() if has_wallet else 0
-        remote_height = self.session.wallet.network.get_server_height() if has_wallet else 0
-        best_hash = (yield self.session.wallet.get_best_blockhash()) if has_wallet else None
-        wallet_is_encrypted = has_wallet and self.session.wallet.wallet and \
+        has_wallet = self.session and self.session.wallet
+        downloading_headers = self.session.wallet.is_downloading_headers() if has_wallet else False
+        has_wallet_with_network = has_wallet and self.session.wallet.network
+        local_height = self.session.wallet.network.get_local_height() if has_wallet_with_network else 0
```

Comment:
If the wallet has not been initialized yet all of these will throw attribute errors. To fix this, reverse the `if`, ie: `local_height = 0 if not has_wallet_with_network else self.session.wallet.network.get_local_height()`